Bottom: 8522c54c73a149499f90c630afa5bbd4d958416b
Top:    2806e54b81615a443bc785d5ebd435a6ba66c82b
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-06 14:36:29 +0800

Refresh of autofs-5.0.5-fix-master-map-bogus-keywork-match.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 1c249ba..e15aa1f 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -46,6 +46,7 @@
 - fix interface address null check.
 - dont probe rdma mounts.
 - fix master map mount options matching.
+- fix master map bogus keywork match.
 
 25/07/2012 autofs-5.0.7
 =======================
@@ -119,7 +120,6 @@
 - mount_nfs.so to honor explicit NFSv4 requests.
 - mount_nfs.so fix port=0 option behavior v3.
 - documentation fix some typos and misleading comments.
-- fix master map bogus keywork match.
 
 28/06/2011 autofs-5.0.6
 -----------------------
@@ -204,7 +204,6 @@
 - add base64 password encode.
 - fix ipv6 name for lookup.
 - fix libtirpc ipv6 check.
-- fix non multi-mount recursive mounts.
 
 03/09/2009 autofs-5.0.5
 -----------------------
